source/mge.js

```diff
@@ -0,0 +1,988 @@
+
+// Experimental
+
+var megengine = megengine || {};
+var python = python || require('./python');
+var base = base || require('./base');
+
+megengine.ModelFactory = class {
```

Comment:
Rename file to `megengine.js` to match namespace.

Reply:
done